### PR TITLE
Capitalize German tags

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Capitalize first character of a tag in German.

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Please review

File: source/blog/_blog-article-meta.erb [hunk 1]
```diff
@@ -10,6 +10,7 @@ author_name = author ? blog_author_name(author) : article.data.author
   </time>
   <span class="post-tags">
     <% article.tags.each do |tag| %>
+      <% tag = tag[0].capitalize + tag[1..-1] if root_locale == :de %>
```
Comment (jessedijkstra) [inline source/blog/_blog-article-meta.erb]: Doesn't `tag.capitalize` work?

Comment (RonaldDijkstra) [inline source/blog/_blog-article-meta.erb]: We use abbreviations like CAPP. When we use capitalize it becomes Capp.

Comment (jessedijkstra) [inline source/blog/_blog-article-meta.erb]: Oh, it removes the capitalisation for existing strings?

Comment (jessedijkstra) [inline source/blog/_blog-article-meta.erb]: Aaah:
```
irb(main):001:0> "CAPP".capitalize
=> "Capp"
irb(main):002:0>
```

Comment (jessedijkstra) [inline source/blog/_blog-article-meta.erb]: Shouldn't the L of "E-learning" be capitalised as well?

screen shot 2017-02-28 at 16 52 18

Comment (RonaldDijkstra) [inline source/blog/_blog-article-meta.erb]: Yup. Maybe I'll just rewrite all the tags and call it a day :)

Comment (fatboypunk) [inline source/blog/_blog-article-meta.erb]: So let's move this to a method
```ruby
def capitalize(tag)
  tag.slice(0,1).capitalize + tag.slice(1..-1)
end
```
and just call capitalize(tag)

**Comment by RonaldDijkstra** [comment]
#490 

Thank you fatboypunk

**Pull request closed without merging**